source/NVDAObjects/behaviors.py

```diff
@@ -366,6 +367,98 @@ def event_loseFocus(self):
 		super(Terminal, self).event_loseFocus()
 		self.stopMonitoring()
 
+
+class TerminalWithKeyboardSupport(Terminal):
+	"""A Terminal object that also provides typed character support for

```

Comment:
Please expand this docstring to cover the following:
- "Provides typed character support by receiving and processing keyboard handler provided events rather than monitoring the console output for changes adjacent to the cursor"
- Explain that it relies on keyboard handler, specifically `toUnicodeEx` and the flag to preserve keyboard state that is only supported in `1607` and above.
- Requires NVDA typed char events
- Processes NVDA typed char events to strip typed passwords, and provide better reporting after tab press.